build-output-api/on-demand-isr/README.md

```diff
@@ -33,6 +33,16 @@ In this demo, you can see this happening with two paths: `/` and `/data`.
 
 When the revalidation is triggered, the cache will be revalidated, bypassing any caching that Vercel would normally provide.
 
+There is also an example of a blog that only generates new paths when triggered via On-Demand ISR with two initial posts that are prerendered.
+
+- `/blog/[slug]`
+  - base post prerender that allows generating news paths via On-Demand ISR
```

Comment:
Is there an API endpoint to trigger this? Or should we document a `fetch()` call example or something?

Reply:
Can add a revalidate endpoint for this similar to the other for clarity.

Reply:
Added in https://github.com/vercel/examples/pull/374/commits/1758bd1dc293f56c5408ff7dbe8c0c04452b2adf